study-builder/studies/snippets/picklist-question-state-required.conf

```diff
@@ -240,6 +240,28 @@
       "allowDetails": false,
       "exclusive": false
     },
+    {
+      "stableId": "DC",
```

Comment:
Updated main state conf file so that any future uses or redeploys will include DC.